specs/resources/funding-sources/retrieveInitiateVerifyMicroDeposit.yml

```diff
@@ -69,7 +69,9 @@ post:
       application/json:
         schema:
           oneOf:
-            - type: [object, null]
+            - type:
+                - 'object'
+                - 'null'
```

Comment:
@ShreyaThapa , do we need to define null or can it be omitted? If we do define null should it be nullable: true as referenced in the spec?

Reply:
I think we'll want both request types to avoid confusion and errors, one for MD initiation and one for Verification. Support for "nullable" was removed in OpenAPI 3.1, so I replaced it with what the [migration doc ](https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0#:~:text=Swap%20nullable%20for%20type%20arrays)suggested.

Reply:
Ah, icic 👍